cpp/core/shuffle/ShuffleWriter.h

```diff
@@ -44,6 +44,10 @@ class ShuffleWriter {
 
   virtual arrow::Status Stop() = 0;
 
+  virtual std::shared_ptr<arrow::Schema>& Schema() {
```

Comment:
Minor hint: we can leave some comments here that this is for write schema and for ```VeloxShuffleWriter``` is difference from ```ArrowShuffleWriter``` since it's serialized as a Arrow array.

Reply:
We will delete `ArrowShuffleWriter` after we completely implement `VeloxShuffleWriter` (velox to velox).